sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/LegacyServiceClient.cs

```diff
@@ -1593,6 +1597,19 @@ private Legacy.JobManifestTasks CreateTasks(TextAnalyticsActions actions)
 
         #endregion
 
+        #region Extract Summary
```

Comment:
Not really related to this PR but I see you're adding this region because `LegacyServiceClient` inherits from `ServiceClient`. I think it's a bit awkward to update this class only with methods that are not supported. Have we already included that in a GA release?